code/game/machinery/cloning.dm

```diff
@@ -234,12 +234,12 @@ GLOBAL_LIST_INIT(cloner_biomass_items, list(\
 		if(!G)
 			return 0
 
-/*
-	if(clonemind.damnation_type) //Can't clone the damned.
-		playsound('sound/hallucinations/veryfar_noise.ogg', 50, 0)
-		malfunction()
- 		return -1 // so that the record gets flushed out
-	*/
+	/*
+	 *	if(clonemind.damnation_type) //Can't clone the damned.
+	 *		playsound('sound/hallucinations/veryfar_noise.ogg', 50, 0)
+	 *		malfunction()
+	 *		return -1 // so that the record gets flushed out
+	 */
```

Comment:
```suggestion
```
this is just commented out code, to the nether realm with it

Reply:
Done